web/client/reducers/queryform.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * Copyright 2015, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const {ADD_FILTER_FIELD, REMOVE_FILTER_FIELD, UPDATE_FILTER_FIELD, UPDATE_EXCEPTION_FIELD} = require('../actions/queryform');
+const assign = require('object-assign');
+
+const initialState = {
```

Comment:
This is the initial state for the example, non of a generic QueryBuilder, the initialState of a generic QueryBuilder should be empty